pkg/mfa/client.go

```diff
@@ -134,7 +154,7 @@ type VerifyOpts struct {
 
 type VerifyResponse struct {
 	// Return details of the request
-	Challenge ChallengeResponse `json:"challenge"`
+	Challenge Challenge `json:"challenge"`
```

Comment:
`VerifyResponse` → `VerifyChallengeResponse` would better match how we named other response types in the SDK